saas/views/download.py

```diff
@@ -409,16 +441,32 @@ class TransferDownloadView(SmartTransactionListMixin,
                            TransferQuerysetMixin, CSVDownloadView):
 
     basename = 'transfers'
+    CSV_CUSTOMER_CONTACTS_INCLUDE = settings.CSV_CUSTOMER_CONTACTS_INCLUDE
+
     headings = [
-        'CreatedAt',
+        'Created At',
         'Amount',
         'Unit',
         'Description'
     ]
 
+    def get_headings(self):
+        if self.CSV_CUSTOMER_CONTACTS_INCLUDE:
+            extra_headings = [
+                'Full Name',
+                'Phone',
```

Comment:
We should also output the 'Email' to be consistent. 'Full Name' most likely should be part of the always in headings.